code/modules/clothing/spacesuits/alien_suits.dm

```diff
@@ -207,12 +207,11 @@
 	species_restricted = list("Vox")
 	sprite_sheets = list(
 		"Vox" = 'icons/mob/clothing/species/vox/feet.dmi')
+	multiple_icons = FALSE
 
 /obj/item/clothing/shoes/magboots/vox/attack_self(mob/user)
 	if(magpulse)
-		flags &= ~NOSLIP
-		magpulse = FALSE
-		flags |= NODROP
```

Comment:
... did I break this?
Anyways, good fixing 👍

Reply:
Seems to be working fine